index.js

```diff
@@ -115,7 +115,13 @@ class SideMenu extends Component {
    */
   handlePanResponderMove(e: Object, gestureState: Object) {
     if (this.state.left.__getValue() * this.menuPositionMultiplier() >= 0) {
-      this.state.left.setValue(this.prevLeft + gestureState.dx);
+      let newLeft = this.prevLeft + gestureState.dx;
+
+      if (!this.props.bounceBackOnOverdraw && Math.abs(newLeft) > this.props.openMenuOffset) {
```

Comment:
I think you're making a wrong conclusions based on what I'm saying. Instead of saying I'm not going to merge this one in, I'm asking you a question to understand what drives you to make this prop. #154 contains only GIF and a phrase "hey, how can I fix it" which is not something I can work with. Seems you understand this problem deeper and that's why I'm asking - I want to learn what you both means by this. Why do you think this miscommunication happened?